ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/clientlibs/editor/js/editDialog.js

```diff
@@ -298,7 +299,7 @@
         channel.on("click", BASE_ENUM_MULTIFIELD_ADD_BUTTON, function (e) {
             let areOptionsRichText = $(BASE_ARE_OPTIONS_RICH_TEXT)[0],
                 enumNames = $(BASE_ENUMNAMES_VISIBLE),
-                richTextEnumNames = $(BASE_WRAPPER_INPUT_RICH_TEXT_ENUMNAMES);
+                richTextEnumNames = $(BASE_RICH_TEXT_ENUMNAMES);
```

Comment:
Please write a test case for this use-case.

Reply:
added